values.yaml

```diff
@@ -85,6 +85,24 @@ st2:
       name: st2packs
       tag: latest
       pullPolicy: Always
+
+    # https://docs.stackstorm.com/reference/ha.html#st2sensorcontainer
+    # It is possible to run st2sensorcontainer in HA mode by running one process on each compute instance.
+    # Each sensor node needs to be provided with proper partition information to share work with other sensor
+    # nodes so that the same sensor does not run on different nodes.
+    sensors:
+      # Default "all-in-one" sensorcontainer that runs all sensors.
+      # To partition sensors with one sensor per node, override st2.packs.sensors.
+      # NOTE: Do not modify this file.
+      - name: aio
+        settings:
```

Comment:
I'll experiment with this. I don't see any reason why not. I'd done it this way because one of the examples I'd seen grouped variables under `settings`. Didn't seem like a bad idea at the time.